drivers/vmwarevsphere/vsphere.go

```diff
@@ -325,9 +326,19 @@ func (d *Driver) PreCreateCheck() error {
 		return err
 	}
 
-	if _, err := f.NetworkOrDefault(ctx, d.Network); err != nil {
-		return err
+	// TODO: if the user has both the VSPHERE_NETWORK defined and adds --vmwarevsphere-network
+	//       both are used at the same time - probably should detect that and remove the one from ENV
+	if len(d.Networks) == 0 {
+		// machine assumes there will be a network
+		d.Networks = append(d.Networks, "VM Network")
+	}
+	for _, netName := range d.Networks {
+		if _, err := f.NetworkOrDefault(ctx, netName); err != nil {
+			return err
+		}
 	}
+	// d.Network needs to remain a string to cope with existing machines :/
+	d.Network = d.Networks[0]
```

Comment:
looking at the output of `machine inspect`, I think its better to keep it compatible and fill it with something - to differentiate from a vm with no networking. (and yup, I'm going to be using no-network vm's in future)